man/bootup.xml

```diff
@@ -92,46 +92,58 @@
-         v                  v                  |                    v                    v
-     (various           (various               |                (various          rescue.service
-    timers...)          paths...)              |               sockets...)               |
-         |                  |                  |                    |                    v
-         v                  v                  |                    v              <emphasis>rescue.target</emphasis>
-   timers.target      paths.target             |             sockets.target
-         |                  |                  |                    |
-         v                  \_________________ | ___________________/
-                                              \|/
-                                               v
-                                         basic.target
-                                               |
-          ____________________________________/|                                 emergency.service
-         /                  |                  |                                         |
-         |                  |                  |                                         v
-         v                  v                  v                                 <emphasis>emergency.target</emphasis>
-     display-        (various system    (various system
- manager.service         services           services)
-         |             required for            |
-         |            graphical UIs)           v
-         |                  |           <emphasis>multi-user.target</emphasis>
-         |                  |                  |
-         \_________________ | _________________/
+<programlisting>
+             cryptsetup-pre.target
+                     |
+                     |
+                     v                              (various low-level
+    (various cryptsetup devices...)                  API VFS mounts:
+                     |    |                          mqueue, configfs,
+                     v    |                          debugfs, ...)
+       cryptsetup.target  |                               |
+                     |    |    remote-fs-pre.target       |  (various swap
+                     |    |     |        |                |   devices...)
+                     |    |     |        v                |    |
+ local-fs-pre.target |    |     |  (network file systems) |    v
+         |           |    v     v                 |       |  swap.target
+         v           |  remote-cryptsetup.target  |       |    |
+(various mounts and  |          |                 |       |    |  (various low-level
+ fsck services...)   |          v       remote-fs.target  |    |   services: udevd,
+         |           |                            |       |    |   tmpfiles, random
+         v           |                            v       |    |   seed, sysctl, ...)
```

Comment:
in this graph i'd move these two to the side and order it before multi-user.target. When enabled these two targets are ordered before that after all, see their [Install] sections.